receiver/sqlserverreceiver/recorders.go

```diff
@@ -76,7 +76,7 @@ var perfCounterRecorders = []perfCounterRecorderConf{
 			"Checkpoint pages/sec": func(mb *metadata.MetricsBuilder, ts pcommon.Timestamp, val float64) {
 				mb.RecordSqlserverPageCheckpointFlushRateDataPoint(ts, val)
 			},
-			"Lazy Writes/sec": func(mb *metadata.MetricsBuilder, ts pcommon.Timestamp, val float64) {
+			"Lazy writes/sec": func(mb *metadata.MetricsBuilder, ts pcommon.Timestamp, val float64) {
```

Comment:
This looks like a bug fix. If so, can you please create an issue and separate PR?

Reply:
I don't think this would be considered a bug because both versions work in collecting the metric value. When the counters are gathered though it is listed as `"Lazy writes`. So `Get-Counter -Counter  "\SQLServer:Buffer Manager\Lazy Writes/sec"` and Get-Counter -Counter  "\SQLServer:Buffer Manager\Lazy writes/sec" commands both return the same value.

Reply:
correct